docs/new-contributors.md

```diff
@@ -0,0 +1,58 @@
+# Introduction
+
+This is a small guide of how to get started with Kubernetes, OpenShift Origins, and KubeVirt. 
+But let's start with how they relate to each other.
+
+*   **Kubernetes** is the upstream container orchestration engine, to run containers on a cluster
+*   **OpenShift** **Origin** is a friendly fork of Kubernetes with additional features, and configured to meet more enterprise requirements (i.e. RBAC authentication) than stock Kubernetes
```

Comment:
Hm, makes sense.  Especially if I consider i.e. tectonic, which is yet another K8s distrobution, then there is no particular reason why it must be mentioned here.

Removing it.